app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,16 @@
+from app.players import player
+
+
+class Dwarf(player.Player):
+    def __init__(self, nickname: str, favourite_dish: str) -> None:
+        super().__init__(nickname)
+        self._favourite_dish = favourite_dish
+
+        def get_rating() -> None:
+            pass
```

Comment:
The `get_rating` method is currently defined without parameters and does not return any value. According to the task requirements, this method should return an integer value representing the player's rating. You should implement this method to return a meaningful rating for the `Dwarf` class.